app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -484,13 +482,13 @@ def parse_server(server, parent_hosts = nil)
       disks = new_result[:hardware][:disks]
       dev = "vda"
 
-      if (sz = flavor[:disk_size]) == 0
+      if (sz = flavor[:root_disk_size]) == 0
```

Comment:
@Fryguy - If you do decide to make PR to change this variable name, you should do the same for the Openstack parser (and any other provider parser that contains the "sz" variable name)